Library/Homebrew/sorbet/rbi/dsl/rubo_cop/cop/formula_audit/assert_statements.rbi

```diff
@@ -0,0 +1,7 @@
+# typed: true
+
+# DO NOT EDIT MANUALLY
+# This is an autogenerated file for dynamic methods in `RuboCop::Cop::FormulaAudit::AssertStatements`.
+# Please instead update this file by running `bin/tapioca dsl RuboCop::Cop::FormulaAudit::AssertStatements`.
+
+class RuboCop::Cop::FormulaAudit::AssertStatements; end
```

Comment:
Note that there are 21 rbi files that are useless that are introduced in this PR. It might be possible to make the dsl compiler logic smarter (I would need a class source API, like the `method_source` gem we rely on for methods) to avoid this. (The initial attempt used a hard-coded [list](https://github.com/Homebrew/brew/pull/16586/commits/d16f469079d1928f080d3f305f4c089a2f8db254#diff-7a7d5e88bed9515000c6906d25ecbc3ebd12edae4d4065c7fea0257c08853f40R18-R43) to avoid this, which I also don't like.)

I think that tapioca should maybe not create rbi files when no action is taken in the decorator, so I might try filing an issue upstream first.